setup.py

```diff
@@ -14,19 +14,13 @@ def read(fname):
 
 
 install_requires = [
-    'mezzanine >= 3.1.10',
     'django-treebeard',
-    'django-filer>=0.9.6',
-    'South',
+    # pyScss fixed until https://github.com/fusionbox/django-pyscss/pull/26 is fixed.
+    'pyScss==1.2.1',
```

Comment:
I'm not very enthusiastic about pinnig versions with `==`. Shouldn't we use `>=1.2,<1.3`?